lib/apipie_bindings/authenticators.rb

```diff
@@ -1,3 +1,4 @@
 require 'apipie_bindings/authenticators/basic_auth'
 require 'apipie_bindings/authenticators/credentials_legacy'
 require 'apipie_bindings/authenticators/oauth'
+require 'apipie_bindings/authenticators/token_auth'
```

Comment:
@mbacovsky the [issue](https://github.com/theforeman/hammer-cli-foreman/pull/405#discussion_r253502324) I was facing this entire time was because of not having his line of code.